stock_storage_type_putaway_abc/readme/DESCRIPTION.rst

```diff
@@ -1,2 +1,17 @@
 This module implements chaotic storage 'ABC' according to Package Storage Type
 and Location Storage Type.
+
+The locations and products get an 'a', 'b' or 'c' storage (by default 'b').
+
+For the computation of the putaway, the locations are sorted first by max
+height which is a physical constain to respect, then by 'ABC' following that:
```

Comment:
```suggestion
height which is a physical constraint to respect, then by 'ABC' as following:
```

Reply:
Thanks. I fixed also the generated README.rst